source/installation/index.rst

```diff
@@ -34,9 +34,38 @@ By default, it will perform a graphical installation, unless you use the msiexec
 Large Installations
 ^^^^^^^^^^^^^^^^^^^
 
+Consider a scenario where the GLPI Agent application needs to be installed on large and diverse range of Windows systems. A VBScript can be useful in this scenario to perform the following tasks:
+
+- Install it silently taking variables from the installation script previously configured.
+- Check for the presence of FusionInventory and OCS Inventory agents and uninstall them if you need it.
+- Configure application settings based on `commands that can be explored here <https://glpi-agent.readthedocs.io/en/latest/installation/windows-command-line.html#command-line-parameters>`.
+- Log installation progress and errors for review and analysis.
+
 A VBScript (Visual Basic Script) is provided to deploy the installer on a network:
 :download:`glpi-agent-deployment.vbs <https://raw.github.com/glpi-project/glpi-agent/develop/contrib/windows/glpi-agent-deployment.vbs>`.
 
+In this script you'll find some variables that can be changed to your environment needings (uncomment what is commented and you need. Comment what you don't need.):
+- **GLPI Agent Version** hereby named ``SetupVersion``
+- Setup the **Location** from where the script will download the MSI hereby named ``SetupLocation``
+ - If it's HTTP, HTTPS, CIFS
+- Setup the **Architecture of your systems** (if you need it to be x86, x64, or if you want to let the it be installed according to the system Architecture) hereby named ``SetupArchitecture``
+- **Setup Options** from the `command line paramenters <https://glpi-agent.readthedocs.io/en/latest/installation/windows-command-line.html#command-line-parameters>`
```

Comment:
use relative path for this

```suggestion
- **Setup Options** from the `command line paramenters <windows-command-line.html#command-line-parameters>`
```

In github preview, I think it doesn't work, but maybe sphinx/readthedocs generate correct links for headings reference (aka document#headings). @g-bougard or @trasher, maybe you an example of usage in some of our documentations ?

Reply:
On my own, in this documentation, I often uses the `:ref:` syntax, like for the `debug` here:
https://github.com/glpi-project/doc-agent/blob/d6eb2ba3a79435a47fb72841e55e9c27a4a11312/source/bug-reporting.rst?plain=1#L29

It needs to create a ref point:
https://github.com/glpi-project/doc-agent/blob/d6eb2ba3a79435a47fb72841e55e9c27a4a11312/source/configuration.rst?plain=1#L304

This has the advantage to leave rtd to resolv itself the right page: it can be moved to another place without requiring you to change all the line referencing it.

Reply:
In this case, you need to insert the following code to the right place:
```
.. _windows-command-line-parameters:
```

And then follow this suggestion:
```suggestion
- **Setup Options** from the :ref:`command line paramenters <windows-command-line-parameters>`
```

Reply:
Otherwise @orthagh suggestion is not working as an underscore is missing:
```suggestion
- **Setup Options** from the `command line parameters <windows-command-line.html#command-line-parameters>`_
```